### PR TITLE
Makefile: Install the iocsh scripts and more sources

### PR DESCRIPTION
**Pull request by tboegi**
In order to load the right version of the start snippets,
install them using the SCRIPTS target.

While at there, use the wildcard *.cpp for SOURCE files.

### CODE REVIEW

**Comment by jeonghanlee** [comment]
@tboegi , please revert the the wildcard *.cpp for SOURCE files to be the original ones. Most cases, it works no differently between them. However, there is a chance when you cannot compile them properly if source codes have *interesting* dependencies, which old environment had one.  I also see EthercatMC has less chances to be, but it would be better to keep the specific compiling order.

File: EthercatMC.Makefile [hunk 1]
```diff
@@ -43,13 +43,11 @@ USR_INCLUDES += -I$(where_am_I)$(APPSRC)
...
 
-SOURCES += $(APPSRC)/EthercatMCAxis.cpp
-SOURCES += $(APPSRC)/EthercatMCController.cpp 
-SOURCES += $(APPSRC)/EthercatMCHelper.cpp 
+SOURCES += $(wildcard $(APPSRC)/*.cpp)
 
 DBDS    += $(APPSRC)/EthercatMcSupport.dbd
 
-
+SCRIPTS += $(wildcard ./iocsh/*.iocsh)
```
Comment (jeonghanlee) [inline EthercatMC.Makefile]: SCRIPTS += $(wildcard ../iocsh/*.iocsh)

File: EthercatMC.Makefile [hunk 2]
```diff
@@ -43,13 +43,11 @@ USR_INCLUDES += -I$(where_am_I)$(APPSRC)
 
 TEMPLATES += $(wildcard $(APPDB)/*.template)
 
-SOURCES += $(APPSRC)/EthercatMCAxis.cpp
-SOURCES += $(APPSRC)/EthercatMCController.cpp 
-SOURCES += $(APPSRC)/EthercatMCHelper.cpp 
+SOURCES += $(wildcard $(APPSRC)/*.cpp)
```
Comment (jeonghanlee) [inline EthercatMC.Makefile]: Revert them to the original codes.

File: EthercatMC.Makefile (hunk 1 above)
Comment (jeonghanlee) [reply EthercatMC.Makefile]: However, I don't see any iocsh files in your pull requests.

**Comment by tboegi** [comment]
Thanks for the review -
I need to push a new EthercatMC (with *.iocsh) and other changes first.

**Pull request merged**